test/test-cases.setup.js

```diff
@@ -0,0 +1,40 @@
+import { basename, dirname, join } from 'path';
+import webpack from 'webpack';
+import I18nPlugin from '../src';
+
+export default function processFile(entry, ...pluginOpts) {
+  const resolvedEntry = join(__dirname, 'test-cases', entry);
+  const resolvedOutput = join(dirname(resolvedEntry), `${basename(resolvedEntry, '.code.js')}.tmp.js`);
```

Comment:
I use it to strip the file's extension properly. 
The equivalent transformation can be `resolvedEntry.replace('.code.js', '.tmp.js');`

Presumably, we can get rid off the `*.tmp.js` files with memoryFs, but It was a challenge for to debug it in the past :)